source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -1205,13 +1205,13 @@ export class TextBlock extends BaseTextBlock {
                 anchor.classList.add(hostConfig.makeCssClassName("ac-anchor"));
                 anchor.target = "_blank";
                 anchor.onclick = (e: MouseEvent) => {
-                    if (raiseAnchorClickedEvent(this, e.target as HTMLAnchorElement, e)) {
+                    if (raiseAnchorClickedEvent(this, anchor, e)) {
```

Comment:
@paulcam206 do we need to treat this as an breaking change for npm package?

Reply:
no, this is just a straight up bug -- if folks are already working around this by walking the parent chain until they see a `<a>`, this change won't break them...